tests/MongoDB.Driver.Core.Tests/Core/Operations/FindCommandOperationTests.cs

```diff
@@ -818,6 +821,34 @@ public void CreateCommand_should_throw_when_ReadConcern_is_set_but_not_supported
             result.Should().Be(expectedResult);
         }
 
+        [SkippableFact]
+        public void CreateCursor_should_use_ns_field_instead_of_namespace_passed_in_constructor()
+        {
+            var subject = new FindCommandOperation<BsonDocument>(_collectionNamespace, BsonDocumentSerializer.Instance, _messageEncoderSettings);
+            var expectedCollectionNamespace = "cursors.lkajlkasdf-3980238d908sdf";
```

Comment:
Refactor to:

   var cursorCollectionNamespace = CollectionNamespace.FromFullName("cursors.lkajlkasdf-3980238d908sdf");

Reply:
Done.